docs/index.html

```diff
@@ -70,16 +64,81 @@ <h3>Example 3</h3>
   &lt;body&gt;
     &lt;script&gt;
       bootstrapGrist({
-        initialFile: 'investment-research.grist',   // path to .grist
-        name: 'Investment Research',                // optional name
+        initialFile: 'weresheet.grist',   // optional path to .grist
+        name: 'Werewolf Fact Sheet',      // optional name
+        elementId: 'were-div',            // optional element name
+        light: true                       // optional minimalist style
       });
     &lt;/script&gt;
   &lt;/body&gt;
-&lt;/html&gt;
-      </code></pre>
-      <script>hljs.highlightAll();</script>
+&lt;/html&gt;</code></pre>
+      <button id="were-click" class="btn btn-primary">Run this code</button>
     </div>
   </div>
+
+  <div class="row mb-5">
+    <div class="col-sm-8">
+      <div class="mb-3">
+        We are experimenting with supporting other formats such as
+        CSV. You can specify `initialData` and have a CSV file loaded,
+        with types autodetected.  This is currently slow, since the
+        full Grist data engine must load before you see anything at
+        all, and we haven't optimized that process yet. Watch this
+        space.
+      </div>
+      <pre id="csv-div"><code class="language-html">&lt;!doctype html&gt;
+&lt;html&gt;
+  &lt;head&gt;
+    &lt;meta charset=&quot;utf8&quot;&gt;
+    &lt;script src=&quot;https://grist-static.com/latest.js&quot;&gt;&lt;/script&gt;
+  &lt;/head&gt;
+  &lt;body&gt;
+    &lt;script&gt;
+      bootstrapGrist({
+        initialData: 'products.csv',
+        light: true
+      });
+    &lt;/script&gt;
+  &lt;/body&gt;
+&lt;/html&gt;</code></pre>
+      <button id="csv-click" class="btn btn-primary">Run this code</button>
+    </div>
+  </div>
+
 </div>
+
+<script>
+  hljs.highlightAll();
+  document.getElementById('were-click').addEventListener('click', function() {
+    const div = document.getElementById('were-div');
+    div.style.height = '500px';
+    div.style.padding = '0';
+    div.style.border = 'none';
+    div.style.overflow = 'hidden';
+    bootstrapGrist({
+      initialFile: 'weresheet.grist',
+      name: 'Werewolf Fact Sheet',
+      elementId: 'were-div',
+      light: true
+    });
+    document.getElementById('were-click').style.display = 'none';
+    return false;
+  });
+  document.getElementById('csv-click').addEventListener('click', function() {
+    const div = document.getElementById('csv-div');
+    div.style.height = '500px';
+    div.style.padding = '0';
+    div.style.border = '1px solid gray';
```

Comment:
Why is this different from the were-click button?

Reply:
The embeds look better without an extra border, but this one is very slow to start, and a border makes it work better with the "watch this space" directive - and gives a little sense that something may be coming eventually.